bin/main.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+'''Copyright (C) 2017 Collabora
```

Comment:
The usual way for adding headers is to use comment blocks (#), not multiline strings.